### PR TITLE
Reduce duplicates in indexes.

### PR DESCRIPTION
**Pull request by oliverchang**
To keep index sizes small.

### CODE REVIEW

File: lib/osv/models.py [hunk 1]
```diff
@@ -331,25 +331,31 @@ def _pre_put_hook(self):
         affected_package.package.purl = purl_helpers.package_to_purl(
             affected_package.package.ecosystem, affected_package.package.name)
 
-    self.project = [
+    self.project = list(set([
```
Comment (inferno-chromium) [inline lib/osv/models.py:334]: nit: sorted(set( would avoid list and later .sort call ?

**Review by inferno-chromium — APPROVED** [review]
LGTM with minor change to simplify

File: lib/osv/models.py (hunk 1 above)
Comment (oliverchang) [reply lib/osv/models.py]: Just trying to avoid a list copy and doing an in-place .sort() instead.

Comment (inferno-chromium) [reply lib/osv/models.py]: smart, i keep forgetting python is a memory hog!

**Review by jonathanmetzman — COMMENTED** [review]
LGTM

**Pull request merged**